### PR TITLE
Rename pkg/deploy -> pkg/apps

### PR DESCRIPTION
**Pull request by soltysh**
@deads2k @mfojtik like promised
@tnozicka fyi

### CODE REVIEW

**Comment by deads2k** [comment]
/lgtm

@soltysh not sure what order you want commits to merge, feel free to bump this one up the queue if you want it before your others.

**Comment by soltysh** [comment]
Updated the missing bits from make verify.

**Comment by soltysh** [comment]
Rebased.

File: pkg/api/install/install.go [hunk 1]
```diff
@@ -37,19 +37,19 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	watchapi "k8s.io/apimachinery/pkg/watch"
 
+	deployapi "github.com/openshift/origin/pkg/apps/apis/apps"
```
Comment (mfojtik) [inline pkg/api/install/install.go]: appsapi

File: pkg/api/install/install.go [hunk 2]
```diff
@@ -37,19 +37,19 @@ import (
...
 	buildapi "github.com/openshift/origin/pkg/build/apis/build"
-	deployapi "github.com/openshift/origin/pkg/deploy/apis/apps"
 	imageapi "github.com/openshift/origin/pkg/image/apis/image"
 	oauthapi "github.com/openshift/origin/pkg/oauth/apis/oauth"
 	projectapi "github.com/openshift/origin/pkg/project/apis/project"
 	routeapi "github.com/openshift/origin/pkg/route/apis/route"
 	templateapi "github.com/openshift/origin/pkg/template/apis/template"
 	userapi "github.com/openshift/origin/pkg/user/apis/user"
 
+	deployv1 "github.com/openshift/origin/pkg/apps/apis/apps/v1"
```
Comment (mfojtik) [inline pkg/api/install/install.go]: appsv1

File: pkg/api/kubegraph/analysis/hpa.go [hunk 3]
```diff
@@ -15,8 +15,8 @@ import (
 	"github.com/openshift/origin/pkg/api/kubegraph"
 	kubeedges "github.com/openshift/origin/pkg/api/kubegraph"
 	kubenodes "github.com/openshift/origin/pkg/api/kubegraph/nodes"
-	deploygraph "github.com/openshift/origin/pkg/deploy/graph"
-	deploynodes "github.com/openshift/origin/pkg/deploy/graph/nodes"
+	deploygraph "github.com/openshift/origin/pkg/apps/graph"
```
Comment (mfojtik) [inline pkg/api/kubegraph/analysis/hpa.go]: appsgraph

File: pkg/api/kubegraph/analysis/hpa_test.go [hunk 4]
```diff
@@ -7,7 +7,7 @@ import (
 	osgraph "github.com/openshift/origin/pkg/api/graph"
 	osgraphtest "github.com/openshift/origin/pkg/api/graph/test"
 	"github.com/openshift/origin/pkg/api/kubegraph"
-	deploygraph "github.com/openshift/origin/pkg/deploy/graph"
+	deploygraph "github.com/openshift/origin/pkg/apps/graph"
```
Comment (mfojtik) [inline pkg/api/kubegraph/analysis/hpa_test.go]: appsgraph

File: pkg/api/kubegraph/edge_test.go [hunk 5]
```diff
@@ -16,9 +16,9 @@ import (
 
 	osgraph "github.com/openshift/origin/pkg/api/graph"
 	kubegraph "github.com/openshift/origin/pkg/api/kubegraph/nodes"
-	deployapi "github.com/openshift/origin/pkg/deploy/apis/apps"
-	_ "github.com/openshift/origin/pkg/deploy/apis/apps/install"
-	deploygraph "github.com/openshift/origin/pkg/deploy/graph/nodes"
+	deployapi "github.com/openshift/origin/pkg/apps/apis/apps"
```
Comment (mfojtik) [inline pkg/api/kubegraph/edge_test.go]: same here

File: pkg/api/kubegraph/edges.go [hunk 6]
```diff
@@ -14,8 +14,8 @@ import (
 
 	osgraph "github.com/openshift/origin/pkg/api/graph"
 	kubegraph "github.com/openshift/origin/pkg/api/kubegraph/nodes"
-	deployapi "github.com/openshift/origin/pkg/deploy/apis/apps"
-	deploygraph "github.com/openshift/origin/pkg/deploy/graph/nodes"
+	deployapi "github.com/openshift/origin/pkg/apps/apis/apps"
```
Comment (mfojtik) [inline pkg/api/kubegraph/edges.go]: and here

**Comment by mfojtik** [comment]
@soltysh mind creating issue to rename the named imports ? s/deployapi/appsapi/ && friends

**Comment by soltysh** [comment]
I'll do a followup, it's much more work with the rename you're asking for and I wanted to have the basic part in, already.

**Pull request merged**